examples/images_contours_and_fields/bivariate_demo.py

```diff
@@ -0,0 +1,40 @@
+"""
+===========
+Bivariate Demo
+===========
+
+Plotting bivariate data.
+
+imshow, pcolor, pcolormesh, pcolorfast allows you to plot bivariate data
+using a bivaraite colormap.
+
+In this example we use imshow to plot air temperature with surface pressure
+alongwith a color square.
+"""
+import matplotlib.colors as colors
+from matplotlib.cbook import get_sample_data
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+###############################################################################
+# Bivariate plotting demo
+# -----------------------
+
+air_temp = np.load(get_sample_data('air_temperature.npy'))
```

Comment:
How big are these data files?  I have a slight preference for 'generated' data for examples (to keep the repository size small).

Reply:
84.2 kb each.
Couldn't come up with generated data so used these.

Reply:
That isn't too bad, suspect that is smaller than the test images.

Reply:
One argument in favour of this can be that the user seeing the example might relate to use case of bivariate plotting better with a real world example than with some mathematical function.